index.html

```diff
@@ -100,50 +169,7 @@ <h4>AnchorCredential</h4>
                 "issuer": "https://sally.example.com/services/anchor",
                 "issuanceDate": "2021-01-27T09:30:10Z",
                 "credentialSubject": {
-                  "type": "AnchorEvent",
```

Comment:
Should we keep these expanded examples everywhere or just condense to the type?

Reply:
If they are the same it is ok to keep it in one spot, reference to it would be nice if possible.